lib/rucio/core/topology.py

```diff
@@ -65,12 +64,13 @@ def __init__(
     @read_session
     def create_from_config(cls, ignore_availability: bool = False, *, session: "Session", logger: "LoggerFunction" = logging.log):
 
-        include_multihop = core_config_get('transfers', 'use_multihop', default=False, expiration_time=600, session=session)
+        include_multihop = config_get('transfers', 'use_multihop', default=False, expiration_time=600, session=session)
+        multihop_rse_expression = config_get('transfers', 'multihop_rse_expression', default='available_for_multihop=true', expiration_time=600, session=session)
 
         multihop_rses = set()
         if include_multihop:
             try:
-                multihop_rses = {rse['id'] for rse in parse_expression('available_for_multihop=true', session=session)}
+                multihop_rses = {rse['id'] for rse in parse_expression(multihop_rse_expression, session=session)}
             except InvalidRSEExpression:
                 pass
```

Comment:
If `include_multihop` is `True` and the RSE expression evaluates to an empty set, wouldn’t it be prudent to have a warning-level log entry?

Reply:
no. But I don't intend to keep that configuration : https://github.com/rucio/rucio/issues/6164 it's redundant

Reply:
Even so, while it doesn’t have to be placed here specifically, I would find it highly useful to have it *somewhere*. For example, when the Preparer starts, log something like ‘Multi-hop is not available’?

Reply:
Done